concourse/execute-test-runs/script.bash

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -e -u
```

Comment:
Would it be a good time to introduce a shell script linter?

[cnx-recipes](https://github.com/openstax/cnx-recipes) uses [shellcheck](https://github.com/koalaman/shellcheck) which is installed on travis containers

Reply:
sure, i'm not super familiar with shell scripting best practices, any tools would be helpful.

in general i've been choosing whatever is the easiest language for these scripts, i have some bash, some js, and some ruby. i like that concourse lets you easily choose the best tool for the job but it does make code convention a little messy... open to thoughts...